bazel/http-server/server.ts

```diff
@@ -34,6 +34,9 @@ export class HttpServer {
     server: {
       directory: false,
       middleware: [(req, res) => this._bazelMiddleware(req, res)],
+      serveStaticOptions: {
+        dotfiles: 'allow',
+      },
```

Comment:
This option seems like a noop since we use a custom middleware anyway, and not the static serve one. Can you remove this please? (unless you see it being needed?)

Reply:
Quickly ran some tests, but you're certainly right. Removed it. 👍

Reply:
thank you!